apigw-sqs-lambda-ddb/README.md

```diff
@@ -0,0 +1,75 @@
+# Amazon API Gateway to Amazon SQS to AWS Lambda to Amazon DynamoDB
+
+This pattern explains how to deploy a SAM application with Amazon API Gateway, Amazon SQS, AWS Lambda, and Amazon DynamoDB. When an HTTP POST request is made to the Amazon API Gateway endpoint, Gateway authorizes the request by checking Basic auth credentials and on valid credentials, request payload is sent to Amazon Simple Queue Service. AWS Lambda function consumes event from the Queue and inserts the event/payload into the Amazon DynamoDB table. Amazon Simple Queue Service is also configured with a Dead Letter Queue where events are sent when retries to process those messages are repeatedly failed.
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any AWS costs incurred. No warranty is implied in this example.
```

Comment:
this is a great pattern and very useful, so i think it would be good to mention in the intro of this pattern why is useful.

Reply:
Expanded the intro section along with some key benefits